src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -74,16 +74,37 @@ $card-left-offset: 40px;
 
     .bcs-task-status {
         align-items: center;
+        color: $bdl-neutral-02;
         display: flex;
-        margin-top: 18px;
+        font-weight: bold;
+
+        > span {
```

Comment:
Is there some way we could avoid these brittle selectors? Maybe adding a className onto the status components?

Reply:
I'm not sure if there's a good way to avoid this; `> span` is to refer to the span that is created for `<FormattedMessage />`, which doesn't take in a class name unfortunately. Do you know if there's a better way to do this?

Reply:
I can probably update the selector `> *:first-child` to be more specific by adding a class though as per your suggestion.

Reply:
You could wrap the FormattedMessage in another span with a className, or pass a function as child: https://github.com/yahoo/react-intl/wiki/Components#formattedmessage

Reply:
Looks doing the function way works for me 💯